conans/client/generators/cmake_find_package.py

```diff
@@ -60,13 +61,15 @@ def _find_for_dep(self, name, cpp_info):
         find_package_header_block = find_package_header.format(name=name, version=cpp_info.version)
         find_libraries_block = target_template.format(name=name, deps=deps, build_type_suffix="")
         target_props = assign_target_properties.format(name=name, deps=deps)
+        contents = []
         tmp = self.template.format(name=name, deps=deps,
                                    version=cpp_info.version,
                                    find_dependencies_block="\n".join(lines),
                                    find_libraries_block=find_libraries_block,
                                    find_package_header_block=find_package_header_block,
                                    assign_target_properties_block=target_props)
-        return tmp
+        contents.append(tmp)
+        return "\n".join(contents)
```

Comment:
This is a bit weird, contents will have only 1 element, right? why the list and the join?

Reply:
yes, this is a leftover from the previous approach